src/script/builder/tir/var.cc

```diff
@@ -27,6 +27,28 @@ tvm::tir::Buffer Buffer_(Array<PrimExpr> shape, DataType dtype, String name, Str
   return tvm::tir::decl_buffer(shape, dtype, name, storage_scope);
 }
 
+tvm::tir::Buffer DeclBuffer(Array<PrimExpr> shape, DataType dtype, String buffer_name,
+                            Optional<tvm::tir::Var> data, Array<PrimExpr> strides,
+                            PrimExpr elem_offset, String storage_scope, int align,
+                            int offset_factor, String buffer_type_str,
+                            Array<IntImm> axis_separators, Span span) {
+  using namespace tvm::tir;
```

Comment:
would you like to check the similarity between this one and `decl_buffer` used in TIR? https://github.com/apache/tvm/blob/main/src/tir/ir/buffer.cc#L50

Reply:
oh okay! i didn't check out but will trust your technical judgment!